libr/magic/mdump.c

```diff
@@ -45,7 +45,7 @@
 
 #ifndef COMPILE_ONLY
 void file_mdump(struct r_magic *m) {
-	static R_TH_LOCAL const char optyp[] = { FILE_OPS };
+	static const char optyp[] = { FILE_OPS };
```

Comment:
keep the R_TH_LOCAL. thats a global
```suggestion
	static R_TH_LOCAL const char optyp[] = { FILE_OPS };
```